ajax/libs/intl-messageformat/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "intl-messageformat",
+  "filename": "intl-messageformat.min.js",
+  "version": "2.2.0",
+  "description": "Formats ICU Message strings with number, date, plural, and select placeholders to create localized messages.",
+  "keywords": [
+    "i18n",
+    "intl",
+    "internationalization",
+    "localization",
+    "globalization",
+    "messageformat",
+    "parser",
+    "plural",
+    "icu"
+  ],
+  "author": "Eric Ferraiuolo <eferraiuolo@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/yahoo/intl-messageformat.git"
+  },
+  "homepage": "https://formatjs.io/",
+  "npmName": "intl-messageformat",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "locale-data",
```

Comment:
Don't use `locale-data` as `basePath`, let's use
```js
{
  "basePath": "",
  "files": [
    "locale-data/*"
  ]
}
```

In this case, the file structure can be consistent.